builtin_interfaces/Time header_stamp
TrajectoryPoint[] points
int32 total_count
